src/mmrelay/meshtastic_utils.py

```
@@ -407,7 +435,12 @@ def connect_meshtastic(passed_config=None, force_connect=False):
                     subscribed_to_connection_lost = True
                     logger.debug("Subscribed to meshtastic.connection.lost")
 
-        except (TimeoutError, ConnectionRefusedError, MemoryError) as e:
+        except (
+            FuturesTimeoutError,
+            TimeoutError,
+            ConnectionRefusedError,
+            MemoryError,
+        ) as e:
             # Handle critical errors that should not be retried
             logger.exception(f"Critical connection error: {e}")
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)